src/server/routes/adminPanel.tsx

```diff
@@ -52,8 +52,9 @@ router.get('/', auth.isAuthenticated, auth.isAuthorized(ADMIN), async (req, res,
 	const {orm} = req.app.locals;
 	const query = parseQuery(req.url).get('q') ?? '';
```

Comment:
```suggestion
	const urlQuery = parseQuery(req.url);
	const query = urlQuery.get('q') ?? '';
```
Here's one missed occurrence of `parseQuery` which can be combined with the next one.

Reply:
fixed!